### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Xiphe** [rehookify/datepicker#27]
Hi there. Thanks a lot for this great and fresh approach! 

Correct me when I'm wrong but it seems this library does not aim to be a drop-in replacement for `<input type="date" />` but instead be a building block that one _could_ use when building a custom datepicker while also being usable for when a datepicker might be rendered inline etc.

Still I think both approaches should be accessible to screen-readers right? 

Also currently I would need to implement keyboard support on my own (which is not that big of a deal thanks to the actions) but I noticed that if I were to add the [three month range example](https://codesandbox.io/p/sandbox/range-with-context-hooks-fyv72j?file=%2Fsrc%2Fapp.tsx%3A1%2C1) inline to a page, as a keyboard user I would have to hit `[Tab]` really often until I eventually land on the next focusable element after the datepicker.

If that's all stuff you think should be done outside of this library that would also be fine for me, just wondering what's your take on this.

Let me know what you think

**Comment by Feshchenko** [comment]
@Xiphe, hi.

Accessibility is super important, we have added aria props to all buttons with this aim.
Keyboard functionality is a bit trickier and demands much more work to cover all cases :)

It is really nice enhancement for the future developement

**Comment by Feshchenko** [comment]
@Xiphe hey.
I know that it has been a while since this issue came up, but I didn't forget about it.
In [v.6.5.0](https://github.com/rehookify/datepicker/releases/tag/v6.5.0) I changed the way prop-getters generates tabIndex.
So now we have 1 `tabIndex: 0` for days, months, years and time.
This issue is partially resolved. 
Next up is to add prop-getters for keyboard interactions for days, months, years and time.
